cypress/integration/data-import/match-on-vendor-reference-number.spec.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable cypress/no-unnecessary-waiting */
+import uuid from 'uuid';
 import getRandomPostfix from '../../support/utils/stringTools';
 import TestTypes from '../../support/dictionary/testTypes';
 import Helper from '../../support/fragments/finance/financeHelper';
```

Comment:
Please fix linter issue:
'FileManager' is defined but never used. Allowed unused vars must match /React/u

Reply:
Please fix linter issue:
'Users' is defined but never used. Allowed unused vars must match /React/u         
'InventoryInstance' is defined but never used. Allowed unused vars must match /React/u   
'MatchProfiles' is defined but never used. Allowed unused vars must match /React/u       
'ActionProfiles' is defined but never used. Allowed unused vars must match /React/u      
'FieldMappingProfiles' is defined but never used. Allowed unused vars must match /React/u

Reply:
Done